operators/src/adapters/raster_overlap_adapter.rs

```diff
@@ -282,21 +325,200 @@ pub fn insert_projected_pixels<'a, T: Pixel, I: Iterator<Item = &'a (GridIdx2D,
     Ok(())
 }
 
+/// This trait defines the behavior of the `RasterOverlapAdapter`.
+pub trait TileSubQuery<T>: Send
```

Comment:
```suggestion
pub trait SubQueryTileAggregator<T>: Send
```
You basically make a subquery and aggregate it to a tile.

Reply:
0d2adeb